src/test/mochitest/browser_dbg-sourcemaps-reload.js

```diff
@@ -53,7 +53,6 @@ add_task(async function() {
   await reload(dbg);
   await waitForSource(dbg, "v1");
   await syncBp;
```

Comment:
wow why is that not needed?

Reply:
From my debugging, it seems that the source is already selected, so the `await` would just wait forever, causing the timeout. It also doesn't seem to be necessary for the last assessment of the number of breakpoints.

Reply:
how does it pass now?

also if it is already selected it should pass immediately

Reply:
Well it passes because I removed the line, I guess. It passed without the line removal before these changes.

I don't really see how my change would affect the behavior of the test though. It seems kind of convoluted overall since it's putting code from different files (`v1.js`, `v2.js`, `v3.js`) under the `v1` source.

If `waitForSelectedSource` is supposed to pass immediately if it is already selected then I guess ` return dbg.selectors.hasSymbols(state, source.toJS());` would be hanging forever for some reason.